qiskit/transpiler/passes/scheduling/scheduling/__init__.py

```diff
@@ -12,6 +12,6 @@
 
 """Scheduling pass to assign instruction start time."""
 
-from .asap import ASAPSchedule
-from .alap import ALAPSchedule
+from .asap import ASAPScheduleAnalysis
+from .alap import ALAPScheduleAnalysis
```

Comment:
```suggestion
from .asap import ASAPSchedule, ASAPScheduleAnalysis
from .alap import ALAPSchedule, ALAPScheduleAnalysis
```
Should we preserve the existing exports of `A{S,L}APSchedule`?

Reply:
They should be preserved in the parent directory of this onr. The directory structure is a bit weird now with `qiskit/transpiler/passes/scheduling/scheduling` containing just the new things, and `qiskit/transpiler/passes/scheduling` containing the old legacy classesm

Reply:
I didn't have good naming for `scheduling` sub-group. Indeed the workflow now is `scheduling -> alignment -> padding`. Do you have any suggestion?

Reply:
Perhaps `timing_analysis` then? scheduling analysis sounds bit weird to me. This makes the difference clearer.

Reply:
I can change it, `timing_analysis` sounds good to me. But I'm also fine leaving it as scheduling. It's just a bit confusing for the next few releases while we deprecate and remove the legacy `ALAPSchedule` and `ASAPSchedule`